src/core/TrainSwitch.hpp

```diff
@@ -1,18 +1,16 @@
 #pragma once
-#include <godot_cpp/classes/node.hpp>
 #include "TrainPart.hpp"
 
 
 namespace godot {
     class TrainSwitch : public TrainPart {
             GDCLASS(TrainSwitch, TrainPart)
 
-        private:
```

Comment:
boli mnie usuniecie "private". argumentacja:
- explicit is better than implicit
- readability counts - nie widze "na pierwszy rzut oka" czy to jest public, protected czy private
- przypadkowo ktos doda u gory klasy protected albo public, i zmieni sie klasyfikacja